Calculator.py

```diff
@@ -1,5 +1,5 @@
 def gather_inputs():
-    operation = input("Please type in operation you would like to complete: +, -, *, or /")
+    operation = input("Please type in operation you would like to complete: +, -, *, /, or ^")
```

Comment:
Can we state the new operation first? EX: `"^, +, -, *, or /"`

Reply:
Done